CHANGELOG.rst

```diff
@@ -213,7 +213,7 @@ Features
   ``.wcs`` is accessed) (`#4467 <https://github.com/sunpy/sunpy/pull/4467>`__)
```

Comment:
I gave up on the changelog.

Reply:
yeah let's no do this to the changelog lol